core/repository/api/src/main/resources/org/eclipse/rdf4j/repository/config/memory.ttl

```diff
@@ -15,9 +15,9 @@
       rep:repositoryType "openrdf:SailRepository" ;
       sr:sailImpl [
          sail:sailType "openrdf:MemoryStore" ;
-	 sail:iterationCacheSyncThreshold "{%Query Iteration Cache sync threshold|10000%}";
+         sail:iterationCacheSyncThreshold "{%Query Iteration Cache sync threshold|10000%}";
          ms:persist {%Persist|true|false%} ;
          ms:syncDelay {%Sync delay|0%} ;
-         sb:evaluationStrategyFactory "{%EvaluationStrategyFactory|org.eclipse.rdf4j.query.algebra.evaluation.impl.StrictEvaluationStrategyFactory%}"
+         sb:defaultQueryEvaluationMode "{%Query Evaluation Mode|strict|standard%}"
```

Comment:
Note that I am removing the config for the factory from these templates. The configuration property can still be used in actual repository configs of course, but this removes the prompting for it from the command line (since we no longer need it except in rare cases). 

Also: `strict` is the first option here, which makes it the default choice. If we decide to go for standard as the new default in 5.0 (see GH-4213) these should be swapped.